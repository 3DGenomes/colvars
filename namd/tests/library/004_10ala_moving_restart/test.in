colvarsTrajFrequency 2

colvar {
  name d
  dihedral {
    group1 { atomNumbers 1 }
    group2 { atomNumbers 4 }
    group3 { atomNumbers 10 }
    group4 { atomNumbers 12 }
  }
  outputAppliedForce
}

harmonic {
  colvars d

  centers 135
  targetCenters 495
  targetNumStages 36
  targetNumSteps  833
  forceConstant 10. 
}

