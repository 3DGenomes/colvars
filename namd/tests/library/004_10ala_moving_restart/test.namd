# FILENAMES
set outName             [file rootname [file tail [info script]]]
#set inFileNum           [expr [scan [string range $outName end-1 end] "%d"] - 1]
#set inName              [format "%s%02u" [string range $outName 0 end-2] $inFileNum]

set inName previous


# MD SETUP
timestep          0.5
numSteps		  20


colvars			  on
colvarsConfig	  test.in
colvarsInput  $inName.colvars.state


# INPUT
structure               ../Common/da.psf 
parameters              ../Common/par_all22_prot.inp
paraTypeCharmm          on
coordinates             ../Common/da.pdb

bincoordinates       $inName.coor
binvelocities        $inName.vel

# OUTPUT

outputname              ${outName}
outputenergies          2000
outputtiming            20000
binaryoutput            yes
restartname             ${outName}r
restartfreq             40000
binaryrestart           yes


seed		87654321

# CONSTANT-T
langevin                on
langevinTemp		    300.0
langevinDamping         10.0
langevinHydrogen	    off

# SPACE PARTITIONING
splitpatch              hydrogen
hgroupcutoff            2.8
stepspercycle           20
margin                  1.0

# CUT-OFFS
switching               on
switchdist              9.0
cutoff                  11.0
pairlistdist            12.0

# RESPA 
fullElectFrequency      1
nonbondedFreq           1

# 1-4 NON-BONDED
exclude                 scaled1-4
1-4scaling              1.0

# COM
commotion               no

# SHAKE
rigidBonds              none

