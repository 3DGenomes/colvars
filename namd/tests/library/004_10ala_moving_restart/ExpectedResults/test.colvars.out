colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 2
colvars: # colvarsRestartFrequency = 40000 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = d
colvars:   Initializing a new "dihedral" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:       Initializing atom group "group1".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group1" defined, 1 atoms initialized: total mass = 14.007, total charge = -0.62.
colvars:       Initializing atom group "group2".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group2" defined, 1 atoms initialized: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group3".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group3" defined, 1 atoms initialized: total mass = 12.011, total charge = 0.51.
colvars:       Initializing atom group "group4".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group4" defined, 1 atoms initialized: total mass = 14.007, total charge = -0.47.
colvars:   All components initialized.
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # upperWallConstant = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { d }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 10
colvars:   # centers = { 135 }
colvars:   # targetCenters = { 495 }
colvars:   # targetForceConstant = 0 [default]
colvars:   # targetNumSteps = 833
colvars:   # targetNumStages = 36
colvars:   # targetForceExponent = 1 [default]
colvars:   # outputCenters = off [default]
colvars:   # outputAccumulatedWork = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Restarting from file "previous.colvars.state".
colvars:   Restarting collective variable "d" from value: -137.21
colvars:   Restarting restraint bias "harmonic1".
colvars:   # centers = { -135 }
colvars:   # centers_raw = { 225 }
colvars:   # stage = 9
colvars: ----------------------------------------------------------------------
colvars: The restart output state file will be "testr.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars:   Moving restraint "harmonic1" stage 10 : setting centers to { -125 } at step 8330
colvars: Saving collective variables state to "test.colvars.state".
